server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreBaseIntegTestCase.java

```diff
@@ -137,7 +137,7 @@ protected BulkResponse indexBulk(String indexName, int numDocs) {
         for (int i = 0; i < numDocs; i++) {
             final IndexRequest request = client().prepareIndex(indexName)
                 .setId(UUIDs.randomBase64UUID())
-                .setSource(documentKeys.get(randomIntBetween(0, documentKeys.size() - 1)), randomAlphaOfLength(5))
+                .setSource("fixedKeyName", randomAlphaOfLength(5))
```

Comment:
Why is this change required?

Reply:
this will be reverted once we have integration with download flow. currently remote index metadata provider we hardcoded mapping